### ISSUE TITLE
Bug: Error Loading Mods - shielding.lua insulated-cable

### ISSUE THREAD

**Issue by andrewmeyer** [Pezzawinkle/PezsMods#23]
![2024-05-19_14-52](https://github.com/Pezzawinkle/PezsMods/assets/3374647/c1f4ad50-caaf-4caa-a7e6-b2589364d629)

This started in a fairly recent update, though I still crash in:
1.1.107
1.1.104
1.1.94
1.1.91

**Comment by andrewmeyer** [comment]
This looks like a regression in 1.0.13.  rolling back to 1.0.12 resolved the issue.
```
core 0.0.0
base 1.1.107
WireShortcuts 1.1.13
aai-signal-transmission 0.4.9
alien-biomes-hr-terrain 0.6.1
artillery-bombardment-remote 1.1.1
better-air-filtering 0.3.6
bigtruck 1.0.0
bobinserters 1.2.0
boblibrary 1.2.0
Bottleneck 0.11.7
deadlock-beltboxes-loaders 2.4.2
EpicArtillerySounds 1.1.1
expanded-rocket-payloads-continued 1.1.1
factorissimo-2-notnotmelon 1.2.3
flib 0.14.1
FluidMustFlow 1.3.5
FNEI 0.4.2
GameStats 1.1.4
HelicopterRevival 0.2.2
Honk 5.0.0
ImprovedCombinatorSE 1.0.15
informatron 0.3.4
item-incineration 1.2.0
jetpack 0.3.14
Kux-CoreLib 2.11.0
Lightning 1.0.15
nixie-tubes 1.1.5
Nuclear_Operators_Utilities 1.2.0
RITEG 1.3.4
robot_attrition 0.5.15
shield-projector 0.1.6
Shortcuts-ick 1.1.28
WormAttack 6.0.0
alien-biomes 0.6.8
bobenemies 1.2.0
bobequipment 1.2.1
bobores 1.2.0
combat-mechanics-overhaul 0.6.24
DeadlockStackingForVanilla 1.1.4
ForceFields2 2.3.8
LoaderRedux 1.9.1
PickerAtheneum 1.2.5
RealisticReactors 3.1.5
bobplates 1.2.1
bobtech 1.2.1
Dectorio 0.12.6
PickerPipeTools 1.1.4
rso-mod 6.2.26
bobassembly 1.2.2
bobgreenhouse 1.2.0
boblogistics 1.2.2
bobmining 1.2.1
bobpower 1.2.1
bobrevamp 1.2.1
bobwarfare 1.2.1
BobPipesThroughput 0.1.1
bobvehicleequipment 1.2.0
electricboiler 1.1.0
intermediate-bulk-containers 1.0.0
Kux-OrbitalIonCannon 2.4.1
SpaceMod 1.1.2
Aircraft 1.8.7
angelsrefining 0.12.7
angelsinfiniteores 0.9.11
angelspetrochem 0.9.26
Clowns-Nuclear 1.3.19
angelsaddons-storage 0.0.10
angelssmelting 0.6.23
cargo-ships 0.2.6
DeepMine 0.6.2
angelsbioprocessing 0.7.27
Asteroid_Mining 0.9.15
LogisticTrainNetwork 1.18.6
angelsindustries 0.4.21
angels-smelting-extended 1.0.12
angelsaddons-cab 0.2.9
Clowns-Processing 1.3.23
RealisticFusionPower 1.8.17
reskins-library 2.2.2
Clowns-AngelBob-Nuclear 1.1.21
extendedangels 0.5.10
RealisticFusionWeaponry 1.5.8
reskins-bobs 2.2.0
rfp-angels-bridge 1.0.0
ERPCTbaAB 0.2.7
reskins-angels 2.2.1
reskins-compatibility 2.2.0
base 1.1.107
Soundpack-alerts-and-sirens-updated-v1-1 1.0.7
bobinserters 1.2.0
deadlock-beltboxes-loaders 2.4.2
factorissimo-2-notnotmelon 1.2.3
KingsTurret-Shields 1.110.2
Nuclear_Operators_Utilities 1.2.0
Shortcuts-ick 1.1.28
alien-biomes 0.6.8
bobenemies 1.2.0
bobequipment 1.2.1
bobores 1.2.0
RealisticReactors 3.1.5
bobplates 1.2.1
bobtech 1.2.1
Dectorio 0.12.6
Realistic_Heat_Glow 2.1.0
bobassembly 1.2.2
bobgreenhouse 1.2.0
boblogistics 1.2.2
bobmining 1.2.1
bobpower 1.2.1
bobrevamp 1.2.1
bobwarfare 1.2.1
bobvehicleequipment 1.2.0
deadlock-integrations 2.2.0
intermediate-bulk-containers 1.0.0
Kux-OrbitalIonCannon 2.4.1
Aircraft 1.8.7
angelsrefining 0.12.7
angelsinfiniteores 0.9.11
angelspetrochem 0.9.26
Clowns-Nuclear 1.3.19
angelsaddons-storage 0.0.10
angelssmelting 0.6.23
cargo-ships 0.2.6
MushroomCloud-patched 1.0.15
angelsbioprocessing 0.7.27
Asteroid_Mining 0.9.15
angelsindustries 0.4.21
angels-smelting-extended 1.0.12
angelsaddons-cab 0.2.9
Clowns-Processing 1.3.23
RealisticFusionPower 1.8.17
reskins-library 2.2.2
Clowns-AngelBob-Nuclear 1.1.21
extendedangels 0.5.10
RealisticFusionWeaponry 1.5.8
reskins-bobs 2.2.0
DeadlockStackingForBobs 1.2.7
ERPCTbaAB 0.2.7
reskins-angels 2.2.1
reskins-compatibility 2.2.0
WireShortcuts 1.1.13
bobinserters 1.2.0
boblibrary 1.2.0
deadlock-beltboxes-loaders 2.4.2
factorissimo-2-notnotmelon 1.2.3
FluidMustFlow 1.3.5
HelicopterRevival 0.2.2
Honk 5.0.0
ImprovedCombinatorSE 1.0.15
jetpack 0.3.14
Kux-CoreLib 2.11.0
nixie-tubes 1.1.5
RITEG 1.3.4
robot_attrition 0.5.15
Shortcuts-ick 1.1.28
Turbwhine 1.0.2
alien-biomes 0.6.8
combat-mechanics-overhaul 0.6.24
DeadlockStackingForVanilla 1.1.4
LoaderRedux 1.9.1
PickerAtheneum 1.2.5
RealisticReactors 3.1.5
bobplates 1.2.1
bobtech 1.2.1
Dectorio 0.12.6
PickerPipeTools 1.1.4
rso-mod 6.2.26
bobassembly 1.2.2
boblogistics 1.2.2
deadlock-integrations 2.2.0
Kux-OrbitalIonCannon 2.4.1
SpaceMod 1.1.2
angelsrefining 0.12.7
angelspetrochem 0.9.26
Clowns-Nuclear 1.3.19
angelssmelting 0.6.23
cargo-ships 0.2.6
DeepMine 0.6.2
Asteroid_Mining 0.9.15
LogisticTrainNetwork 1.18.6
angelsindustries 0.4.21
RealisticFusionPower 1.8.17
reskins-library 2.2.2
DeadlocksStackingForAngelsSmeltingExtended 0.0.3
extendedangels 0.5.10
RealisticFusionWeaponry 1.5.8
reskins-bobs 2.2.0
DeadlockStackingForBobs 1.2.7
reskins-angels 2.2.1
DeadlockStackingForAngels 1.2.2
DeadlockStackingForMadclown 0.0.9
DeadlockStackingForMadclown_MdRuz 0.0.6
reskins-compatibility 2.2.0
```

**Comment by andrewmeyer** [comment]
confirmed working in 0.14
